packages/blockchain-wallet-v4-frontend/src/data/modules/settings/sagas.js

```diff
@@ -128,7 +128,11 @@ export default ({ coreSagas }) => {
       yield put(actions.alerts.displaySuccess(C.IPRESTRICTION_UPDATE_SUCCESS))
     } catch (e) {
       yield put(actions.logs.logErrorMessage(logLocation, 'updateIpLockOn', e))
-      yield put(actions.alerts.displayError(C.IPRESTRICTION_UPDATE_ERROR))
+      if (e === "You must add at least 1 ip address to the whitelist") {
```

Comment:
I'd add this one as a constant and export it for testing purposes

Reply:
Do you mean create a new service that exports this or just declare and export it in this file @tony-blockchain?

Reply:
for now just in this file, but some constant exporting files would be good ofc